example_notebooks/7_ALS_toyblock_demo.ipynb

```diff
@@ -0,0 +1,1829 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !"run/helios.exe" data/surveys/demo/light_als_toyblocks_multiscanner.xml
</pre><p>Maybe we could run it with the <code>--splitByChannel</code>  argument as well to showcase this feature. </p><p> </p><p>And then down below create three plots side by side which show the outputs of the individual scanners. @sina Do you think you can manage or shall I add this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/3dgeo-heidelberg/helios/pull/309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>I will try my best and ask for assistance if nothing works:)</p>